packages/forma-36-react-components/src/components/Button/README.md

```diff
@@ -0,0 +1,117 @@
+# Button
+
+Buttons communicate the action that will occur when the user clicks them. They communicate calls to action to the user and allow users to interact with pages in a variety of ways. They contain a text label to describe the action, and an icon if appropriate.
+
+## Button types
+
+- **Primary** - Used for the most important actions in any scenario. Don’t use more than one primary button in a section or screen to avoid overwhelming users.
+- **Muted** - Used for a secondary actions, the most commonly used button type.
+- **Positive** - For use when the action has a positive connotation such as creating or publishing a new entity.
+- **Negative** - For destructive actions - when something can't be undone. For example, deleting entities.
+- **Naked** - Link button
+
+
+![button overview](images/img.png)
+
+Buttons can be used in couple of different variations. Worth mentioning is `indicateDropdown` property in the component, which enables chevron icon on the right side of the button. Using `icon` property in the Button on the other hand, enables choosen icon on the left side of the button.
+
+![Button variations](images/img-1.png)
+
+Contentful buttons are available in 3 different sizes: 
+ - large - 46px height
```

Comment:
I'd list only the name of the sizes, or just showcase using code blocks, but I wouldn't add values here